src/popup/router/pages/PermissionsSettings.vue

```diff
@@ -0,0 +1,187 @@
+        {{ $t('pages.permissions.message-sign') }}
+      </p>
+    </ListItem>
+    <ListItem class="permission-row">
+      <p class="text-left">{{ $t('pages.permissions.transaction-sign') }}</p>
+      <Input
+        :value="permissions.transactionSignLimit || ''"
+        :error="error"
+        placeholder="no limit"
+        @input="value => changePermission('transactionSignLimit', value)"
+      />
+    </ListItem>
+    <div>
+      <RangeInput
+        :value="permissions.transactionSignLimit"
+        min="0"
+        :max="tokenBalance"
+        step="0.1"
+        @input="value => changePermission('transactionSignLimit', value)"
+      />
+    </div>
+    <ListItem class="permission-row">
+      <p class="text-left grow">{{ $t('pages.permissions.spent-today') }}</p>
+      <span class="tokens">
+        {{ (permissions.transactionSignLimit - limitLeft).toFixed(2) }}
+        <span class="secondary-text">&nbsp;{{ $t('pages.appVUE.aeid') }}</span>
+      </span>
+      <span class="currency">{{
+        `(${formatCurrency(convertToCurrency(permissions.transactionSignLimit - limitLeft))})`
+      }}</span>
+    </ListItem>
+    <ListItem class="permission-row">
+      <p class="text-left grow">{{ $t('pages.permissions.left-today') }}</p>
+      <span class="tokens">
+        {{ limitLeft.toFixed(2) }}
+        <span class="secondary-text">&nbsp;{{ $t('pages.appVUE.aeid') }}</span>
+      </span>
+      <span class="currency">{{ `(${formatCurrency(convertToCurrency(limitLeft))})` }}</span>
+    </ListItem>
+    <ListItem class="permission-row">
+      <p class="text-left grow">{{ $t('pages.account.balance') }}</p>
+      <span class="tokens">
+        {{ tokenBalance }}
+        <span class="secondary-text">&nbsp;{{ $t('pages.appVUE.aeid') }}</span>
+      </span>
+      <span class="currency">{{ `(${formatCurrency(balanceCurrency)})` }}</span>
+    </ListItem>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import CheckBox from '../components/CheckBox';
+import ListItem from '../components/ListItem';
+import Input from '../components/Input';
+import RangeInput from '../components/RangeInput';
+import { getLimitLeft, setLimitLeft } from '../../../store/modules/permissions';
+
+export default {
+  components: {
+    ListItem,
+    CheckBox,
+    Input,
+    RangeInput,
+  },
+  data: () => ({
+    error: false,
+    limitLeft: 0,
+  }),
+  computed: {
+    ...mapGetters(['tokenBalance', 'convertToCurrency', 'balanceCurrency', 'formatCurrency']),
+    permissions() {
+      return this.$store.state.permissions;
+    },
+  },
+  async mounted() {
+    const storageLimitLeft = (await getLimitLeft())?.limitLeft;
+    this.limitLeft =
+      storageLimitLeft === undefined ? this.permissions.transactionSignLimit : storageLimitLeft;
+  },
+  methods: {
+    async changePermission(name, value) {
+      // eslint-disable-next-line no-restricted-globals
+      if (isNaN(value) || !isFinite(value) || value < 0) {
```

Comment:
Proposing to avoid `eslint-disable` by using Number.isNaN (but it works a bit differently)